defaults/main.yml

```diff
@@ -39,3 +39,7 @@ kube_storage_backend: "etcd3"
 enable_external_admission_controller_webhook: False
 
 kube_apiserver_runtime_config: "batch/v2alpha1=true{% if enable_external_admission_controller_webhook %},admissionregistration.k8s.io/v1alpha1=true{% endif %}"
+
+kube_enable_cloud_provider: False
+kube_cloud_config_path: "/etc/kubernetes/vsphere.conf"
```

Comment:
does this need to be vsphere.conf? cant we call it cloud-config.conf

Reply:
We can give any name to file. Will make it ```cloud-config``` so that it looks generic.